sound/soc/sof/pcm.c

```diff
@@ -673,6 +701,9 @@ int sof_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd, struct snd_pcm_hw_pa
 	/* read rate and channels from topology */
 	switch (dai->dai_config->type) {
 	case SOF_DAI_INTEL_SSP:
+		/* search for config to pcm params match, if not found use default */
+		ssp_dai_config_pcm_params_match(sdev, (char *)rtd->dai_link->name, params);
```

Comment:
you dont check the return value here, so why bother with the return -EINVAL at all in the function?

Reply:
This is for the future, if I need to do comparison with several different match rules/functions, now I'm just ignoring the error -> go with default config